frontend/src/metabase/containers/DataPicker/CardPicker/CardPickerContainer.tsx

```diff
@@ -0,0 +1,176 @@
+/* eslint-disable react/prop-types */
```

Comment:
Would really appreciate some help here. For some reason, eslint goes crazy about `value` and `onChange` props not being listed as props (which it normally parses from a typescript type). Final `CardPickerProps` includes `CardPickerOwnProps` which extends `DataPickerProps` that contains both `value` and `onChange`, but eslint still complaints :/